toshling/_client.py

```diff
@@ -40,7 +47,8 @@ def __init__(self, api_key, api_endpoint_base='https://api2.toshl.com'):
         self.me = endpoints.Me(self)
         self.tags = endpoints.Tags(self)
     
-    def request(self, href, method, argument_type=None, return_type=None, **kwargs):
+    def request(self, href, method, argument_type=None, return_type=None,
+                return_response=False, **kwargs):
```

Comment:
I do wonder if, rather than having an argument dictate the return type(s), it might be better to make this a different method entirely. If you want to get really fancy, you could actually make it a generator and use it directly to iterate over the requests themselves.

Something like `request_pages()` or `iterate_pages()`? Or even just `iterate()`.